GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -123,6 +126,7 @@ private void Done()
                 {
                     UpdateLabel.Text = _noUpdatesFound.Text;
                 }
+                Dispose();
```

Comment:
Why do you dispose the Form? It is supposed to show the `_noUpdatesFound.Text` in case when there are no updates found.